web-api/src/main/java/dk/dbc/opensearch/input/SearchRequest.java

```diff
@@ -0,0 +1,193 @@
+        collapseHitsThreshold = get("collapseHitsThreshold", collapseHitsThreshold, content, location,
+                                    s -> Integer.parseUnsignedInt(trimNotEmpty(s)));
+    }
+
+    public Integer getCollapseHitsThreshold() {
+        return collapseHitsThreshold;
+    }
+
+    public void setCollectionType(String content, Location location) throws XMLStreamException {
+        collectionType = get("collectionType", collectionType, content, location,
+                             COLLECTION_TYPES);
+    }
+
+    public CollectionType getCollectionType() {
+        return collectionType == null ? CollectionType.WORK : collectionType;
+    }
+
+    public void addFacets(Facets content, Location location) throws XMLStreamException {
+        if (facets == null)
+            facets = new ArrayList<>();
+        facets.add(content);
+    }
+
+    public List<Facets> getFacets() {
+        return facets;
+    }
+
+    public void addObjectFormat(String content, Location location) throws XMLStreamException {
+        objectFormat.add(get("objectFormat", content, location, OBJECT_FORMATS));
+    }
+
+    public List<String> getObjectFormats() {
+        return objectFormat.isEmpty() ? Arrays.asList("marcxchange") : objectFormat;
+    }
+
+    public void setQuery(String content, Location location) throws XMLStreamException {
+        query = get("query", query, content, location,
+                    s -> trimNotEmpty(s));
+    }
+
+    public String getQuery() {
+        return query;
+    }
+
+    public void setQueryDebug(String content, Location location) throws XMLStreamException {
+        queryDebug = get("queryDebug", queryDebug, content, location,
+                         s -> Boolean.parseBoolean(trimNotEmpty(s)));
+    }
+
+    public Boolean getQueryDebug() {
+        return queryDebug;
+    }
+
+    public void setQueryLanguage(String content, Location location) throws XMLStreamException {
+        queryLanguage = get("queryLanguage", queryLanguage, content, location,
+                            QUERY_LANGUAGES);
+    }
+
+    public String getQueryLanguage() {
+        return queryLanguage;
+    }
+
+    public void addSort(String content, Location location) throws XMLStreamException {
+        if (userDefinedRanking == null)
+            throw new XMLStreamException("Cannot have both userDefinedRanking and sort", location);
+        if (sort == null)
+            sort = new ArrayList<>();
+        sort.add(get("sort", content, location, s -> trimNotEmptyOneWord(s)));
+    }
+
+    public List<String> getSort() {
+        return sort;
+    }
+
+    public void setStart(String content, Location location) throws XMLStreamException {
+        start = get("start", start, content, location,
+                    s -> Integer.parseUnsignedInt(trimNotEmpty(s)));
+    }
+
+    public Integer getStart() {
+        return start == null ? 0 : start;
```

Comment:
Iflg https://opensource.dbc.dk/services/open-search-web-service er default value for start = 1